docs-archive/apache-airflow-providers-google/0.0.1/index.html

```diff
@@ -0,0 +1 @@
+Placeholder for apache-airflow-providers-google v0.0.1
```

Comment:
Do we have to commit a new file for every provider's version? Can it be automated?

Can you please explain how the process of releasing doc per provider should look like ?

Reply:
Ah I see the PR in Airflow. will review tomorrow :)